src/stripeClient.ts

```diff
@@ -102,46 +102,39 @@ export class StripeClient {
   }
 
   detectInstalled() {
-    const osType: OSType = getOSType();
-    let installPaths: string[] = [];
-
-    switch (osType) {
-      case OSType.macOS:
-        // HomeBrew install path on macOS
-        installPaths = ['/usr/local/bin/stripe'];
-        break;
-      case OSType.linux:
-        // apt-get install path on ubuntu + yum install path on centOS
-        installPaths = ['/usr/local/bin/stripe'];
-        break;
-      case OSType.windows:
-        // scoop install path on Windows 10
-        const userProfile = process.env.USERPROFILE || '';
-        installPaths = [path.join(userProfile, 'scoop', 'shims', 'stripe.exe')];
-        break;
-    }
+    const defaultInstallPath = (() => {
```

Comment:
noob question: why do we define the function and then evaluate and assign here as opposed to doing something like calling this function in line 139?

Reply:
I prefer using immediately invoked functions for assignments like this because it feels "more functional" to me, which I prefer. I feel like either approach is valid though. Do you think moving this function out would be more readable/maintainable?

Reply:
No need, I was asking out of curiosity. Agreed that either approach works.